examples/fargate-serverless/README.md

```diff
@@ -24,7 +24,10 @@ To provision this example:
 
 ```sh
 terraform init
+terraform apply -target module.vpc
```

Comment:
not required - single apply works for this example

Reply:
If you say so. :)